crates/bevy_app/src/app.rs

```diff
@@ -203,67 +212,65 @@ impl App {
         self
     }
 
-    /// Returns the state of all plugins. This is usually called by the event loop, but can be
-    /// useful for situations where you want to use [`App::update`].
-    // TODO: &mut self -> &self
+    /// Update all plugins. This is usually called by the event loop.
```

Comment:
This needs to discuss what "clean" means here. More context about what this does and why it's useful would be good too.